server/src/main/java/io/mantisrx/master/jobcluster/job/MantisJobMetadataImpl.java

```diff
@@ -327,31 +327,31 @@ public URL getJobJarUrl() {
   
    	@Override
 	public Optional<Instant> getStartedAtInstant() {
-		if(this.startedAt == -1 || this.startedAt == 0) {
+		if(this.startedAt == NOT_SET) {
 		    return Optional.empty();
 		} else {
 		    return Optional.of(Instant.ofEpochMilli(startedAt));
 		}
 		
 	}
    	
-   	public long getStartedAt() {
+	public long getStartedAt() {
    	    return this.startedAt;
    	}
 
 	@Override
 	public Optional<Instant> getEndedAtInstant() {
-	    if(this.endedAt == -1) {
+	    if(this.endedAt == NOT_SET) {
```

Comment:
default has changed from `-1` to `0` in couple places, wonder if that might break something. maybe change the check to `this.endedAt <= 0` ?